swaplabel: add page (#10462)
tldr-pages/tldr: pages/linux/swaplabel.md

# swaplabel

> > Set a label for a swap partition.

Reviewer: ```suggestion
> Print or change the label or UUID of a swap area.
> Note: `path/to/file` can either point to a regular file or a swap partition.
```

> > More information: <https://man7.org/linux/man-pages/man8/swaplabel.8.html>.

Reviewer: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links
```suggestion
> More information: <https://manned.org/swaplabel>.
```

> - Set a swap partition label:
> 
> `swaplabel --label "{{new_label}}" /dev/{{sda1}}"`

Reviewer: ```suggestion
- Display the current label and UUID of a swap area:

`swaplabel {{path/to/file}}`

- Set the label of a swap area:

`swaplabel --label {{new_label}} {{path/to/file}}`

- Set the UUID of a swap area (you can generate a UUID using `uuidgen`):

`swaplabel --uuid {{new_uuid}} {{path/to/file}}`
```
